### PR TITLE
Update guides with merged changes from Joel PR

### PR DESCRIPTION
**Pull request by nanaeaubry**
Took edits from Joel's PR to update the GeoEnrichment guides while keeping much of what we already have.

### CODE REVIEW

File: guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb [hunk 1]
```diff
@@ -12,138 +12,250 @@
    ]
```
Comment (CMPeng) [inline guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb:12]: <p>IS this an alternative to using the geo-enrichment source from AGOL? Is so, then we should not use <code>arcgis_pro</code> and <code>arcgis_agol</code> respectively in the rendering and data access statements (in the later cells), instead we might just use a single variable indicating that these two are alternative to each other.</p><p>Also, is Arcgis Pro with BA extension a pre-requisite condition? If so, we might need to state in an <code>Important Note</code> or <code>Pre-requisite</code> highlighting statement??   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by CMPeng** [comment]
@nanaeaubry Thanks for merging the changes - modifications in the first notebook is pretty impressive. Just having two concerns (As put above). Otherwise, looking awesome to me!

**Comment by nanaeaubry** [comment]
@CMPeng 

I added more notebooks to serve as further examples for enrichment

In addition I took note of what you said with agol_gis and pro_gis and made it just gis.
Nothing has changed for using enrich with AGOL. New code was added to use local installs of country datasets and all that. (not touched in the guides because mostly used by GE team)

**Review by CMPeng — APPROVED** [review]
Thanks for making the changes! LGTM :)

**Comment by CMPeng** [comment]
> @CMPeng
> 
> I added more notebooks to serve as further examples for enrichment
> 
> In addition I took note of what you said with agol_gis and pro_gis and made it just gis. Nothing has changed for using enrich with AGOL. New code was added to use local installs of country datasets and all that. (not touched in the guides because mostly used by GE team)
> 
> I need help making the Maps show up however, I run it with maps but they don't show when looking at them in the Notebook reviewer. Any advice for this?

@nanaeaubry Did you try `MapView.take_screenshot()` to create a static embedded image in the notebook. If that does not work, try procedures in https://github.com/Esri/arcgis-python-api/wiki/How-to-embed-map-images-in-your-notebooks. Thanks!

**Comment by nanaeaubry** [comment]
Added Screenshots of maps

**Pull request merged**